llm_env.py

```diff
@@ -0,0 +1,23 @@
+import subprocess
+import sys
+
+def create_virtual_environment(name):
+    # Step 1: 創建虛擬環境
+    subprocess.run([sys.executable, "-m", "venv", name])
+
+    # Step 2: 啟動虛擬環境
+    activate_script = "Scripts\\activate" if sys.platform == "win32" else "bin/activate"
+    activate_path = f"{name}/{activate_script}"
+    subprocess.run([activate_path], shell=True)
+
+    # Step 3: 安裝必要的套件
+    subprocess.run(["pip", "install", "Flask"])
+
+    # Step 4: 安装 Transformers 
+    subprocess.run(["pip", "install", "transformers"])
+
+    print(f"虛擬環境 {name} 創建成功！")
+
+if __name__ == "__main__":
+    venv_name = "llm"
+    create_virtual_environment(venv_name)

```

Comment:
I am curious what is workflow that we use the file. 
Is it like firstly I use the file to construct a `venv` env and activate it, then I can start to use the other LLM files that you submit along the PR? 

I wonder if it is better to have `requirments.txt` instead of this file, or is this file simply a 懶人包, more lazier than the `requirements.txt`? Please leave me a comment here, I hope to understand your thought.